app/controllers/cards_controller.rb

```diff
@@ -10,6 +10,10 @@ class CardsController < AuthenticatedController
   before_action :set_or_initialize_card
   before_action :initialize_sidebar, only: [:show, :new, :edit]
   before_action :set_auto_save_key, only: [:new, :create, :edit, :update]
+  before_action :set_next_item_and_prev_item, only: :move
+  before_action :set_parent, only: :move
+  # Not at top because we need prev item and next item set first
+  include ValidateMove
```

Comment:
```suggestion
  before_action :set_parent, only: :move
  
  # Not at top because we need prev item and next item set first
  include ValidateMove
```